.changeset/large-steaks-film.md

```diff
@@ -0,0 +1,43 @@
+---
+'astro': major
+---
+
+Replace the content collection `slug()` config with a new `slug` frontmatter field.
```

Comment:
Can you specify that this is relevant to people using the previous experimental version in 1.x? New users trying this for the first time don't need to know about this being a breaking change.

Reply:
Done! Clarified this applies to 2.0 beta users as well.